src/main/java/org/recordrobotics/charger/commands/auto/AutoMoveClaw.java

```diff
@@ -58,7 +50,7 @@ public void initialize() {
 	 */
 	@Override
 	public boolean isFinished() {
-		return _claw.getSwitchState() || _claw.getCurrent() > _claw._CURRENT_GRAB_THRESHOLD;
+		return _claw.getSwitchState() || _claw.getPosition() > -0.4;
```

Comment:
this should call `_claw.isReleased()` or `_claw.isGrabbed()`, instead of this object knowing the claw's internal implementation details